mcs/class/System/System.Diagnostics/Process.cs

```diff
@@ -302,7 +302,7 @@ ProcessModule[] GetModules_internal (SafeProcessHandle handle)
 		[DesignerSerializationVisibility (DesignerSerializationVisibility.Hidden)]
 		[MonitoringDescription ("The session ID for this process.")]
 		public int SessionId {
-			get { throw new NotImplementedException (); }
+			get { return 4; }
```

Comment:
Yes. You can check in Task Manager if you enable the Session ID column, a lot of the services run in Session 0.